spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      old_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = old_locale
```

Comment:
If the `expect` raises on failure, then maybe wrap it in a begin/rescue, so we always reset the LANG?